src/bot/service/getMessages.ts

```diff
@@ -24,13 +26,14 @@ export const getMessagesFrom = async ({
         username,
         chatId,
       )) as Message[];
-      messages.forEach(async (message) => {
+      for (const message of messages) {
```

Comment:
Changed because await doesn't work in `forEach` loops so the promises were not being resolved in each iteration